### ISSUE TITLE
Accessibility improvement :  Insufficient colour contrast for the table of content title

### ISSUE THREAD

**Issue by tfrancart** [speced/respec#5339]
Accessibility report have reported an insuficcient contrast of the "Table of Content" title:

<img width="340" height="101" alt="Image" src="https://github.com/user-attachments/assets/eebd1afc-0f8d-43fc-8fb2-4fca7c217f6c" />

Excerpt from the report:

> The h2 title “Table of contents” (font size 22.4px) uses the colour #526b7ae6 (#526b7a with opacity 0.9) on a background colour #f7f8f9. This combination results a contrast ratio of 4.3:1, which fails the WCAG requirement of 4.5:1 for text smaller than 18pt/24px. 
> The text colour for the h2 title must be darkened until the contrast ratio against the white background is at least 4.5:1.
> The “Table of Contents” title has a 90% opacity. By increasing the opacity to 100% (colour #526b7a), we can ensure it meets the required 5.3:1 contrast ratio.

**Comment by marcoscaceres** [comment]
I don't think we own the style for this. This might be a W3C style sheet issue instead?

**Comment by tfrancart** [comment]
> I don't think we own the style for this. This might be a W3C style sheet issue instead?

Thanks, is there a repo somewhere at the W3C where this could be notified ?

**Comment by marcoscaceres** [comment]
@tfrancart that would be https://github.com/w3c/tr-design I think.

**Comment by marcoscaceres** [comment]
Closing as we can't fix it on the ReSpec side.
